client/src/pages/people/Form.js

```diff
@@ -33,9 +33,9 @@ import {
   Alert,
   Button,
   Col,
+  FormCheck,
```

Comment:
We have repetetive Form components. We can simply import Form and use dot notation such as Form.Check, Form.Group, etc...

Reply:
Yes, but I did not want to change more than was absolutely necessary for *just* this change. Harmonizing the imports of `Form*` can be done later (in one go).